nikola/data/themes/base/assets/xml/atom.xsl

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:dc="http://purl.org/dc/elements/1.1/" version="1.0">	
+<xsl:output method="xml"/>
+<xsl:template match="/">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+<head>
+<meta charset="UTF-8"/>
+<meta name="viewport" content="width=device-width"/>
+<title><xsl:value-of select="feed/title"/> (Atom feed)</title>
+<style><![CDATA[html{margin:0;pdding:0;}body{color:hsl(180,1%,31%);font-family:Helvetica,Arial,sans-serif;font-size:17px;line-height:1.4;margin:5%;max-width:35rem;padding:0;}input{min-width:20rem;margin-left:.2rem;padding-left:.2rem;padding-right:.2rem;}ol{list-style-type:disc;padding-left:1rem;}h2{font-size:22px;font-weight:inherit;}]]></style>
+</head>
+<body>
+<h1><xsl:value-of select="feed/title"/> (Atom feed)</h1>
+<p>This is an Atom feed. To subscribe to it, copy its address and paste it when your feed reader asks for it. It will be updated periodically in your reader. New to feeds? <a href="https://duckduckgo.com/?q=how+to+get+started+with+rss+feeds" title="Search on the web to learn more">Learn more</a>.</p>
+<p>
```

Comment:
The search query still says `rss`.

On a side note, could you make it possible to discourage people from using feeds for subpages (index-1 etc.)? I have absolutely no idea how this all works and if this is possible; maybe you could create two XSL files with different text for the two cases.